### ISSUE TITLE
The ImageToolbar class might extracted to ui module as a general class or helper.

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5-ui#400]
A follow up of [table UI](https://github.com/ckeditor/ckeditor5-table/pull/23).

While implementing Table toolbar I've copied most of the `ImageToolbar` code in order to implement it's toolbar. Both classes does practically the same - displays a toolbar attached to some editor element. There are minor differences in details (like obtaining which element to attach the toolbar - AFAIR from util methods rather then class itself) but IMHO 80% code is duplicated - mostly the logic to show/hide or reposition toolbar.

**Comment by oleq** [comment]
I agree, let's do that to save some bytes and fix regressions in one place.
